app.py

```diff
@@ -189,15 +190,26 @@ def login():
 
 @open_id.after_login
 def after_login(resp):
-    flask.session['openid'] = {
-        'identity_url': resp.identity_url,
-        'nickname': resp.nickname,
-        'fullname': resp.fullname,
-        'image': resp.image,
-        'email': resp.email
-    }
-
     flask.session['macaroon_discharge'] = resp.extensions['macaroon'].discharge
+
+    try:
+        account = publisher_api.get_account(flask.session)
+        flask.session['openid'] = {
```

Comment:
I'd say that in this case calling it `openid` is a bit misleading, because there is more information from account API then from openid itself.

So maybe it should be `flask.session['user']` or something similar?